extensions/interactions/guppy.html

```diff
@@ -1,4 +1,9 @@
 <link rel="stylesheet" href="/third_party/static/guppy-b5055b/build/guppy-default.min.css">
+<script>
+  // Use the normal default browser eventlistener without passive set to false.
+  EventTarget.prototype.addEventListener = (
+    EventTarget.prototype.addEventListener._original);
+</script>
```

Comment:
Will this set the event listener only for the guppy library or for everything could you tests if the Best practices pass on some page that uses guppy?

Reply:
It sets the event listener on pages where guppy is loaded because guppy needs to prevent default which is not allowed when passive is set to true.
The best practices metric fails for these pages (exploration editor and exploration player).

Reply:
Okay, lets do it this way for now but add a TODO commet that this should be fixed alter.

Reply:
Done.